src/main/kotlin/day_1_kotlin/solutions/KotlinWorkshop3.kt

```diff
@@ -1,99 +1,183 @@
 package day_1_kotlin.solutions
 
 import java.util.*
+import java.util.logging.Logger
 import kotlin.random.Random
 
 // Workshop #3 - functions, filters, lambda, high-order function
-
 object KotlinWorkshop3 {
 
-    private val guesses = mutableListOf<Int>()
-
+    // You can run the main function to test the code
     @JvmStatic
     fun main(args: Array<String>) {
-        val upperBound = getUpperBound()
+
+        /* Exercise area */
+
+        // Initializing variables in runtime with input.
+        // While running the program. Look at the console. Click inside the console window. Input a number.
+        // Don't touch! Given:
+        val scanner = Scanner(System.`in`)
+        val upperBound = getUpperBound(scanner)
         val nonNullUpperBound = upperBound ?: 10
         val randomNumber = Random.nextInt(nonNullUpperBound)
-        val scanner = Scanner(System.`in`)
+        val capacity = nonNullUpperBound
+        val guesses = createIntArrayOfCapacity(capacity)
 
         var guessed = false
-        while (!guessed) {
-            guessed = playRound(scanner, randomNumber)
+        var index = 0
+        while (!guessed && index < capacity) {
+            print("Input a number in range 0..$nonNullUpperBound inclusive: ")
+            // Storing user input in userInput variable
+            // While running the program. Look at the console. Click inside the console window. Input a number.
+            val userInput: Int = getNextInput(scanner, nonNullUpperBound)
+            guesses[index] = userInput
+
+            // TODO 1: Uncomment. Declare playRound function: takes 2 integer arguments and returns a boolean.
```

Comment:
"Uncomment here and in some other places doesn't makes sense now, as there is nothing to uncomment, right?